package.json

```diff
@@ -46,27 +46,12 @@
   "name": "@hint/monorepo",
   "private": true,
   "scripts": {
-    "build": "yarn clean && yarn build:hint && yarn build:utils && npm-run-all --parallel build:parser:* && yarn build:connectors && yarn build:utils-test && loop \"yarn build\" --cwd packages/ --exclude configuration-development,configuration-progressive-web-apps,configuration-web-recommended,connector-edge,hint,utils-debugging-protocol-common,utils-connector-tools,utils-tests-helpers,connector-chrome,connector-jsdom,connector-local,parser-babel-config,parser-html,parser-javascript,parser-manifest,parser-typescript-config,parser-webpack-config",
-    "build:connector-chrome": "cd packages/connector-chrome && yarn build",
-    "build:connector-edge": "node -e \"process.platform !== 'win32' && process.exit(1);\" && (cd packages/connector-edge && yarn build) || echo \"\"",
-    "build:connector-jsdom": "cd packages/connector-jsdom && yarn build",
-    "build:connector-local": "cd packages/connector-local && yarn build",
-    "build:connectors": "yarn build:connector-chrome && yarn build:connector-jsdom && yarn build:connector-local",
-    "build:hint": "cd packages/hint && yarn build",
-    "build:parser:babel": "cd packages/parser-babel-config && yarn build",
-    "build:parser:html": "cd packages/parser-html && yarn build",
-    "build:parser:js": "cd packages/parser-javascript && yarn build",
-    "build:parser:manifest": "cd packages/parser-manifest && yarn build",
-    "build:parser:ts": "cd packages/parser-typescript-config && yarn build",
-    "build:parser:webpack": "cd packages/parser-webpack-config && yarn build",
+    "build": "yarn clean && yarn build:assets && yarn build:ts",
```

Comment:
Maybe this should be `yarn clean && yarn update:references && ...`  to ensure that everything is up-to-date?

Reply:
Sounds good to me.